volume/volume_test.go

```diff
@@ -0,0 +1,130 @@
+package volume
+
+import (
+	"runtime"
+	"strings"
+	"testing"
+)
+
+func TestParseMountSpec(t *testing.T) {
+	var (
+		valid   []string
+		invalid map[string]string
+	)
+
+	if runtime.GOOS == "windows" {
+		valid = []string{
+			`d:\`,
+			`d:`,
+			`d:\path`,
+			`d:\path with space`,
+			`d:\pathandmode:rw`,
+			// TODO Windows post TP4 - readonly support `d:\pathandmode:ro`,
+			`c:\:d:\`,
+			`c:\windows\:d:`,
```

Comment:
Thanks! (I wasn't sure about the `\\?\`; we talked about it, but didn't remember if it was or was not supported yet :-))